pkgs/tools/security/sherlock/default.nix

```
@@ -2,18 +2,19 @@
 , fetchFromGitHub
 , makeWrapper
 , python3
+, unstableGitUpdater
 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "sherlock";
-  version = "unstable-2023-10-06";
+  version = "unstable-2024-05-12";
   format = "other";
 
   src = fetchFromGitHub {
     owner = "sherlock-project";
     repo = pname;
```

```suggestion
    repo = "sherlock";
```
Being explicit is better here. Else if for some reason, pname changes, it'll break the whole derivation.